controls/cis_ocp_1_4_0/section-2.yml

```diff
@@ -0,0 +1,79 @@
+controls:
+- id: '2'
+  title: etcd
+  status: pending
+  rules: []
+  controls:
+  - id: '2.1'
+    title: Ensure that the --cert-file and --key-file arguments are set as appropriate
```

Comment:
For some reason - the draft XLSX has duplicate rows for level 1 and level 2 controls. I'm working on this with CIS directly.

Reply:
Thanks, but this patch looks fine.